p2p/simulations/network.go

```diff
@@ -210,6 +221,24 @@ func (self *Conn) event(up, rev bool) *ConnEvent {
 	}
 }
 
+type Msg struct {
+	One   *adapters.NodeId `json:"one"`
+	Other *adapters.NodeId `json:"other"`
+	Code  uint64
+}
+
+func (self *Msg) String() string {
+	return fmt.Sprintf("Msg(%d) %v->%v", self.Code, self.One.Label(), self.Other.Label())
+}
+
+func (self *Msg) event() *MsgEvent {
```

Comment:
Action and Type for consistency, I would arge. At least for now (since we're using this cytoscape.go file).

MsgEvent already contains senderid and receiverid through the msg-member..?